### PR TITLE
changed server-client connection

### PR DESCRIPTION
**Pull request by maucon**

### CODE REVIEW

File: src/main/kotlin/de/twintorx/battleship/connection/Server.kt [hunk 1]
```diff
@@ -20,46 +21,44 @@ class Server(
...
+        Writer.println("${ServerMessage.LOCAL_ADDRESS} ${InetAddress.getLocalHost().hostAddress}" +
                 "\n${ServerMessage.PUBLIC_ADDRESS} ${try {
                     BufferedReader(InputStreamReader(URL("http://checkip.amazonaws.com").openStream())).readLine()
                 } catch (e: Exception) {
                     "-"
                 }}")
 
+        Writer.println("\n${ServerMessage.WAITING_PLAYER1}")
+
+        // Host connecting
```
Comment (DennisGeb) [inline src/main/kotlin/de/twintorx/battleship/connection/Server.kt:37]: change to player 1 connecting

File: src/main/kotlin/de/twintorx/battleship/connection/Server.kt [hunk 2]
```diff
@@ -20,46 +21,44 @@ class Server(
...
+        clientSockets[false] = ClientSocket(server.accept().also {
             Writer.println("${ServerMessage.PLAYER2_CONNECTED}${it.inetAddress.hostAddress}")
-        }
-        clientSockets[false] = Triple(client2, Scanner(client2.getInputStream()),
-                PrintWriter(OutputStreamWriter(client2.getOutputStream()), true))
+        })
 
         // Sending start signal to clients
-        clientSockets.values.forEach { it.third.println("1") }
+        clientSockets.values.forEach { it.write(Package(1)) }
```
Comment (DennisGeb) [inline src/main/kotlin/de/twintorx/battleship/connection/Server.kt:50]: change to Package()

File: src/main/kotlin/de/twintorx/battleship/connection/Server.kt [hunk 3]
```diff
@@ -69,7 +68,7 @@ class Server(
         // draw starting player
         val startingPlayer = Random.nextBoolean()
         for ((key, value) in clientSockets) {
-            value.third.println(if (key == startingPlayer) "1" else "0") // sending player starting
+            value.write(Package(if (key == startingPlayer) 1 else 0)) // sending player starting
```
Comment (DennisGeb) [inline src/main/kotlin/de/twintorx/battleship/connection/Server.kt:71]: send true and false

**Comment by TheSwerik** [comment]
Morris is such a noob

**Pull request merged**